pages/introduction.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <section class="h-full px-9 pb-8 overflow-hidden" :class="{'bg-spotify-green': step === 0}">
+    <transition
+        mode="out-in"
+        :name="transition">
+      <div v-if="step === 0" class="pt-24 flex flex-col h-full justify-between">
+        <h1 class="text-4xl leading-[2.75rem] max-w-[12ch] font-bold">Hi, <br/>Welcome to what am I listening to.</h1>
+        <button @click="nextStep" class="bg-spotify-black py-4 rounded-full font-bold text-base inline-flex items-center justify-center gap-[0.875rem]">
+          <span>Continue</span>
+          <ArrowRight class="w-4 aspect-square"/>
+        </button>
+      </div>
+      <div v-else-if="step === 1" class="mx-2 relative text-center grid place-content-center h-full">
+        <div>
+          <img src="../assets/images/step-1.png" class="rounded-lg max-h-[288px] mx-auto"/>
+          <h1 class="mt-9 mb-4 font-bold text-2xl">I'm always trying to find out more about myself</h1>
+          <p>what better window into your <br/>emotions is there than your listening habits?</p>
+        </div>
+        <button @click="nextStep" class="bg-spotify-green absolute bottom-0 w-full py-4 rounded-full font-bold text-base inline-flex items-center justify-center gap-[0.875rem]">
+          <span>Continue</span>
+          <ArrowRight class="w-4 aspect-square"/>
+        </button>
+      </div>
+      <div v-else-if="step === 2" class="mx-2 relative text-center grid place-content-center h-full">
+        <div>
+          <img src="../assets/images/step-2.png" class="rounded-lg max-h-[288px] mx-auto"/>
+          <h1 class="mt-9 mb-4 font-bold text-2xl">Why don't you give a try? Analyse yourself.</h1>
+        </div>
+        <div class="absolute bottom-0 w-full flex items-center gap-4">
+          <button class="bg-gray p-3 rounded-full" @click="previousStep">
+            <ArrowRight class="rotate-180 transform w-6 aspect-square"/>
+          </button>
+          <button @click="routeToExploration" class="bg-spotify-green w-full py-4 rounded-full font-bold text-base inline-flex items-center justify-center gap-[0.875rem]">
+            Explore your taste
+          </button>
+        </div>
+      </div>
+    </transition>
+  </section>
+</template>
+
+<script setup lang="ts">
+import ArrowRight from "~/assets/svg/arrow-right.svg?component"
+
+const step = useState<number>(() => 0)
+const transition = useState<string>(() => 'slide-next')
+
+const previousStep = () => {
+  transition.value = 'slide-previous'
```

Comment:
Is the ".value" necessary here? I'm new to Nuxt and left it out when writing the [newest component](https://github.com/konstantinlob/What-am-I-listening-to/blob/master/components/TimeframeSelection/TimeframeSelection.vue). It doesn't seem to be necessary when referencing it in the template at least.

Reply:
You are right. In the template it is not necessary because the values are already hydrated. In <script setup> you have a Computed Reference to the values when using the Reactivity API with `reactive` or `ref` or `useState` (which is basically `ref` but SSR compatible). See Vue Reactivity API: https://vuejs.org/api/reactivity-core.html